tests/test_fixtures.py

```diff
@@ -98,3 +98,44 @@ def create_an_ic_manifest(root_dir='', n_images=2, n_categories=3):
         coco_path = pathlib.Path(root_dir) / 'coco.json'
         coco_path.write_text(json.dumps(coco_dict))
         return CocoManifestAdaptorFactory.create(DatasetTypes.IMAGE_CLASSIFICATION_MULTICLASS).create_dataset_manifest(coco_path.name, root_dir)
+
+
+class MultilabelClassificationTestFixtures:
+    DATASET_INFO_DICT = {
+        "name": "dummy",
+        "version": 1,
+        "type": "image_classification_multilabel",
+        "root_folder": "dummy",
+        "format": "coco",
+        "test": {
+            "index_path": "train.json",
+            "files_for_local_usage": [
+                "train.zip"
+            ]
+        },
+    }
+
+    @staticmethod
+    def create_an_ic_dataset(n_images=2, n_categories=3):
+        dataset_dict = copy.deepcopy(MultilabelClassificationTestFixtures.DATASET_INFO_DICT)
+        tempdir = tempfile.TemporaryDirectory()
```

Comment:
This TemporaryDirectory() is not removed. Maybe change this method to context manager with `@contextlib.contextmanager`?

Reply:
thanks, added `with tempdir` statement when calling this function so the tempdir will be automatically deleted